app/src/main/java/com/example/movies/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.movies
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.movies.databinding.FragmentMoviesDetailsBinding
+
+class FragmentMoviesDetails : Fragment(R.layout.fragment_movies_details) {
+    private var _binding:FragmentMoviesDetailsBinding? = null
+    private val binding get() = _binding!!
```

Comment:
apply
https://github.com/kirich1409/ViewBindingPropertyDelegate

and use onViewCreated instead of onCreateView